src/components/BusinessForm/BusinessForm.jsx

```diff
@@ -82,345 +75,671 @@ const BusinessForm = ({ pending, pendingData }) => {
   const [city, setCity] = useState('');
   const [state, setState] = useState('');
   const [zip, setZip] = useState('');
-  const [country, setCountry] = useState('');
   const [website, setWebsite] = useState('');
+  const [countrycode, setCountryCode] = useState('');
   const [phone, setPhone] = useState('');
   const [email, setEmail] = useState('');
   const [howHeard, setHowHeard] = useState('');
-  const [registrationSuccess, setRegistrationSuccess] = useState(false);
-
-  const [businessHours, setBusinessHours] = useState(
-    daysOfWeek.reduce(
-      (acc, day) => ({
-        ...acc,
-        [day]: {
-          start: { hour: '9', minute: '00', ampm: 'AM' },
-          end: { hour: '5', minute: '00', ampm: 'PM' },
-        },
-      }),
-      {},
-    ),
-  );
+  const navigate = useNavigate();
+  const [businessHours, setBusinessHours] = useState('');
+  const [checkedAddedInfo, setCheckedAddedInfo] = useState({});
+  const [webNotes, setWebNotes] = useState('');
+  const [published, setPublished] = useState(false);
+  const [webDateInit, setWebDateInit] = useState(new Date());
+  const [internalNotes, setInternalNotes] = useState('');
+  const [fphPhone, setFPHPhone] = useState('');
+  const [vendorType, setVendorType] = useState('');
+  const [serviceRequest, setServiceRequest] = useState(false);
+  const [isAdminUser, setIsAdminUser] = useState(false);
+  const [termsAndConditionsIsOpened, setTermsAndConditionsIsOpened] = useState(false);
+  const [termsAndConditionsAccepted, setTermsAndConditionsAccepted] = useState(false);
+  const [updatedBy, setUpdatedBy] = useState(null);
+  const { backend } = useBackend();
+  const [notification, setNotification] = useState([]);
 
   useEffect(() => {
-    if (pending) {
-      fillOutPendingData();
+    const checkIsAdmin = async () => {
+      if (!(await isAdmin())) {
+        navigate('/BusinessDashboard');
+      } else {
+        setIsAdminUser(true);
+      }
+    };
+
+    checkIsAdmin();
+
+    const fetchNotifications = async () => {
+      const response = await backend.get('/notification/0');
+      setNotification(response.data);
     }
-  });
 
-  const handleTimeChange = (day, period, value) => {
-    setBusinessHours({ ...businessHours, [day]: { ...businessHours[day], [period]: value } });
-  };
+    if (notification.length === 0) {
+      fetchNotifications();
+    }
 
-  const handleZipChange = e => {
-    setZip(e.target.value);
-  };
+    if (edit) {
+      fillOutPendingData();
+    }
+
+    const getAdminUser = async () => {
+      try {
+        const adminUserResponse = await backend.get(`adminUser/${currentUser.email}`);
+        setUpdatedBy(adminUserResponse.data[0].name);
+      } catch (error) {
+        console.log(error);
+      }
+    }
+    if (currentUser) {
+      getAdminUser();
+    }
+  }, [backend]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has missing dependencies: 'currentUser', 'edit', 'fillOutPendingData', 'isAdmin', 'navigate', and 'notification.length'. Either include them or remove the dependency array.